src/el/geth/geth_launcher.star

```diff
@@ -51,10 +51,6 @@ def launch(
     participant_index,
     network_params,
 ):
```

Comment:
This is where the launch method refactor described in `1.` takes place. (same for cl and vc launch methods)